src/xmltv.js

```diff
@@ -1,12 +1,15 @@
 const XMLWriter = require('xml-writer')
 const fs = require('fs')
+const CacheImageService = require('./cache-image-service');
+let dataBase = undefined;
```

Comment:
Is this variable being used outside of the `WriteXMLTV` function? If not, would we be able to remove this line and use `db` in `WriteXMLTV`, keeping it scoped just to that function?